packages/medusa-file-minio/src/services/minio.js

```diff
@@ -1,11 +1,11 @@
-import fs from "fs"
-import aws from "aws-sdk"
 import { AbstractFileService } from '@medusajs/medusa'
+import aws from "aws-sdk"
+import fs from "fs"
 
 class MinioService extends AbstractFileService {
   
   constructor({}, options) {
-    super()
+    super({}, options)
```

Comment:
thought: it doesn't really make sense to pass `options` down as `options` will be plugin-specific and not related to anything in the core right?

Reply:
```suggestion
    super({})
```

Reply:
We'll have to pass it in order for us to use `fileService.withTransaction.someMethod` otherwise the cloned file service will not have plugin options

Reply:
Ah makes sense! :)

Reply:
Should have added this to the description. Will do now 👍